Fill.py

```diff
@@ -835,49 +837,35 @@ def swap_location_item(location_1: Location, location_2: Location, check_locked:
     location_2.item.location = location_2
 
 
-def distribute_planned(multiworld: MultiWorld) -> None:
+def parse_planned_blocks(multiworld: MultiWorld) -> typing.Dict[int, typing.List[typing.Dict[str, typing.Any]]]:
```

Comment:
With these return values getting so complicated, it might be nice to start using type aliases or something